assets/src/data/eventespresso/core/resolvers/relations.js

```diff
@@ -67,8 +67,9 @@ export function* getRelatedEntities(
 	if ( entity.isNew ) {
 		return DEFAULT_EMPTY_ARRAY;
 	}
-	const modelName = entity.modelName.toLowerCase();
+	relationModelName = singularModelName( relationModelName );
 	const pluralRelationName = pluralModelName( relationModelName );
```

Comment:
was this missed? we don't want plurals anymore right? or are they still used in some places?

Reply:
No not missed.  In this case its needed to extract the relation resource from the endpoint (see two lines after this).